ocaml/lib/snarky_js_bindings_lib.ml

```diff
@@ -2133,7 +2187,9 @@ module Choices = struct
               raise_errorf
                 "Rules array is sparse; the entry at index %i is missing" index )
         in
-        let (Rule rule) = Inductive_rule.create ~public_input_size js_rule in
+        let (Rule rule) =
+          Inductive_rule.create ~public_input_size ~public_output_size js_rule
```

Comment:
just a question: What is a rule? Is it a "branch" of our program? (like a single method within the larger contract?)

Reply:
yes, "rule" = method, also called "branches" or "choices" in other places :D